--- a/functions/_tide_item_status.fish
+++ b/functions/_tide_item_status.fish
@@ -5 +5 @@ function _tide_item_status
-                tide_status_color=$tide_status_color_failure _tide_print_item status $tide_status_icon_failure' ' 1
+                tide_status_color=$tide_status_color_failure _tide_print_item status 1' ' $tide_status_icon_failure
@@ -8 +8 @@ function _tide_item_status
-            test $_tide_status = 0 && _tide_print_item status $tide_status_icon' ' $out ||
+            test $_tide_status = 0 && _tide_print_item status $out' ' $tide_status_icon ||
@@ -10 +10 @@ function _tide_item_status
-                    _tide_print_item status $tide_status_icon_failure' ' $out
+                    _tide_print_item status $out' ' $tide_status_icon_failure
